### PR TITLE
es6-modules - Simon Kim.

### PR DESCRIPTION
**Pull request by Skim-329**
es6-modules

### CODE REVIEW

File: es6-modules/src/lib/create-element.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const toArray = require('./to-array');
```
Comment (scbowler) [inline es6-modules/src/lib/create-element.js:1]: This should be an import statement

File: es6-modules/src/lib/create-element.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  toArray(children).forEach(child => {
+    if (!(child instanceof HTMLElement)) {
+      child = document.createTextNode(child);
+    }
+    parent.appendChild(child);
+  });
+  return parent;
+}
+
+module.exports = createElement;
```
Comment (scbowler) [inline es6-modules/src/lib/create-element.js:17]: This should be an export statement

File: es6-modules/src/lib/noop.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+function noop() {}
+
+module.exports = noop;
```
Comment (scbowler) [inline es6-modules/src/lib/noop.js:3]: This should be an export statement

File: es6-modules/src/lib/to-array.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+function toArray(value) {
+  return Array.isArray(value) ? value : [value];
+}
+
+module.exports = toArray;
```
Comment (scbowler) [inline es6-modules/src/lib/to-array.js:5]: This should be an export statement

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but a few files were missed. Let me know if you have any questions

**Comment by Skim-329** [comment]
it was too good to be true, lol. All fixed!

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**